pages/api/users/[id]/attendance/[class_id].ts

```diff
@@ -0,0 +1,35 @@
+import { NextApiHandler, NextApiRequest, NextApiResponse } from "next";
+import { StatusCodes } from "http-status-codes";
+import { getSingleUserAttendanceFromClassID } from "../../../../../lib/database/attendance";
+
+export const userAttendanceHandler: NextApiHandler = async (req: NextApiRequest, res: NextApiResponse) =>{
```

Comment:
lets add a comment at the top saying exactly what this does. Something like "gets all attendance information for a user given a class"